build_latest.sh

```diff
@@ -293,6 +302,11 @@ function build_dockerfile {
 # Set the OSes that will be built on based on the current arch
 set_arch_os
 
+# Updating `oses` for `test` runtype to reduce the build time for OpenJ9 images for PR checks
+if [ "${runtype}" == "test" ] && [ "${vm}" == "openj9" ] && [ "${current_arch}" == "x86_64" ] && [ "${os_family}" == "linux" ]; then
+    oses="${PR_TEST_OSES}"
```

Comment:
Looks like this is using spaces, needs to be tab

Reply:
Changed to use tab.